Tasks/Anevich_Kristina/HW04/hw_04.py

```diff
@@ -0,0 +1,45 @@
+
+while True:
+    user = input("Введите длину строки больше 35:\n")
+    if user.isdigit():
+        user = int(user)
+        if user < 35:
+            print("Длина строки должна быть больше 35 символов! Повторите попытку.\n")
+            continue
+        else:
+            break
+    else:
+        print("Пожалуйста, введите цифру!\n")
+        continue
+
+with open("text.txt", "r", encoding = 'utf-8') as f:
+    text = f.readlines()
+    print(text)
```

Comment:
reading the whole file in memory is not the best approach, since it can be pretty huge